core/storage/improvements/gae_models_test.py

```diff
@@ -91,7 +96,12 @@ def test_export_data_with_task(self):
             resolver_id='uid')
         self.assertEqual(
             improvements_models.TaskEntryModel.export_data('uid'),
```

Comment:
Probably should add a test with 2 items.

Reply:
done